curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/6690bd5ad96139157eedf697.md

```diff
@@ -25,70 +25,68 @@ The `h1` element, `h2` element and the comment are indented two spaces more than
 Your code should have an `h2` element with text `Cat Photos`. You may have accidentally deleted it, it is missing both opening and closing tags, or the text has changed.
 
 ```js
-assert(
-  document.querySelector('h2') &&
-    code.match(/<\/h2\>/) &&
-    document.querySelector('h2').innerText.toLowerCase() === 'cat photos'
-);
+assert.exists(document.querySelector('h2'));
+assert.match(code, /<\/h2\>/);
+assert.equal(document.querySelector('h2').innerText.toLowerCase(),'cat photos')
 ```
 
 You should not add the `ul` or `li` elements from the example.
 
 ```js
-assert(
+assert.isTrue(
```

Comment:
can you make this 2 separete asserts using `assert.notExists`? it makes more sense with what is tested